packages/preact/src/index.ts

```diff
@@ -152,7 +152,18 @@ hook(OptionsTypes.DIFF, (old, vnode) => {
 				// first Signal prop triggers creation/cleanup of the updater:
 				if (!updater) updater = getElementUpdater(vnode);
 				// track which props are Signals for precise updates:
-				updater._props.push(i);
+				updater._props.push({ _key: i, _signal: value });
+				let newUpdater = updater._updater
+				if (value._updater) {
+					let oldUpdater = value._updater
+					value._updater = () => {
+						newUpdater();
+						oldUpdater();
+					}
```

Comment:
I think this might create infinite updaters. If we're setting `_props`, how come we need to run the old updater?

Reply:
Signals reused fot multi attributes test 3 I added

Reply:
Sorry, I meant when vnodes change. In X, `vnode` is unique every time this hook is called.